src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -564,8 +564,8 @@ export default {
 		// Extract query
 		this.pageQuery = _get(this.$route, 'query');
 
-		// Ensure page offset gets set before loading cached data
-		this.updateFromParams(this.pageQuery);
+		// Reset pagination when loading the page
+		this.resetPagination();
```

Comment:
Since this method triggers pushing to the router, I don't think this is the complete solution. Based on a conversation in Slack, we might want to pause on this changeset, if we are later moving all filters to the query string.